qcelemental/models/results.py

```diff
@@ -347,3 +348,18 @@ def _wavefunction_protocol(cls, value, values):
             return ret_wfn
         else:
             return wfn
+
+    @validator('stdout', pre=True)
+    def _logfile_protocol(cls, value, values):
```

Comment:
Does this one have to be a `pre`? Sorry, I forget exactly why the others needed to be.

Reply:
I didn’t think so when I looked up pre but erred on the side of matching other protocol. Will try w/o.